Frigate Camera Notifications/Beta

```diff
@@ -1235,15 +1253,28 @@ action:
                                       required states: {{input_states}}, 
                                       TEST: {{'PASS' if state_true else 'FAIL'}}, 
                                     Custom Filter: {{'PASS' if custom_filter else 'FAIL'}},
+                                    Custom_filter: {{custom_filter}},
+                                    Home: {{home}},
+                                    Zone_filter: {{zone_filter}},
+                                    zone_multi_filter:{{zone_multi_filter}},
+                                    zone_multi_ex_filter:{{zone_multi_ex_filter}},
+                                    state_true: {{state_true}},
+                                    new_snapshot:{{new_snapshot}},
+                                    presence_changed: {{presence_changed}},
+                                    stationary_moved: {{stationary_moved}},
+                                    zone_only_changed: {{zone_only_changed}},
+                                    entered_zones_changed:{{entered_zones_changed}},
+                                    sub_label_changed:{{sub_label_changed}},
+                                    Test Condition loop: "{{ loitering or (custom_filter and not home and zone_filter and (zone_multi_filter or zone_multi_ex_filter) and state_true and (new_snapshot or presence_changed or stationary_moved or zone_only_changed or entered_zones_changed or sub_label_changed)) }}",
```

Comment:
leave for now but probably remove this once testing is done.

Reply:
yeap that was the plan ;-)